deploy/import-upstream.sh

```diff
@@ -1,2 +1,14 @@
-oc import-image coreos/prometheus-operator:v0.25.0 --from=quay.io/coreos/prometheus-operator:v0.25.0 --confirm
-oc import-image coreos/configmap-reload:v0.0.1 --from=quay.io/coreos/configmap-reload:v0.0.1 --confirm
+#!/bin/sh
+oc import-image prometheus:v2.11.104 --from=quay.io/prometheus/prometheus --confirm
+oc import-image prometheus-operator:latest --from=quay.io/coreos/prometheus-operator:v0.27.0 --confirm
```

Comment:
Our downstream version is 0.25.0. I assume there is nothing wrong with 0.27.0 but I wonder if we want to sync across for a semi-matching experience between downstream and upstream?

Reply:
Resolved.